src/app/core/services/auth.service.ts

```diff
@@ -33,6 +33,9 @@ export enum AuthState {
  * updating the application state with regards to authentication.
  */
 export class AuthService {
+  private static readonly DEFAULT_HAS_PRIVATE_PERMISSION = true;
```

Comment:
Not sure if this should be the default. See #214.

Reply:
I have changed this to `false`. This means that auto-redirect does not work if the user paste the link to view a private repo. There needs extra work to handle viewing private repos, I will create a separate issue.